src/include/migraphx/builtin.hpp

```diff
@@ -90,7 +90,12 @@ struct param
 struct returns
 {
     std::string name() const { return "@return"; }
-    shape compute_shape(const std::vector<shape>&) const { return {}; }
+    shape compute_shape(const std::vector<shape>& arg) const
+    {
+        if(arg.size() > 1)
+            return shape(arg);
+        return arg[0];
```

Comment:
if `arg` is  empty it would seg fault.

Reply:
Agreed. Thanks. This check was inadvertently removed in code reorg. Good catch.